upgradeability_using_generic_eternal_storage/test/TokenProxy.js

```diff
@@ -3,9 +3,9 @@ const assertRevert = require('./helpers/assertRevert')
 
 const Token_V0 = artifacts.require('Token_V0')
 const Token_V1 = artifacts.require('Token_V1')
-const TokenProxy = artifacts.require('TokenProxy')
+const EternalStorageProxy = artifacts.require('EternalStorageProxy')
 
-contract('TokenProxy with generic Eternal Storage', ([_, proxyOwner, tokenOwner, anotherAccount]) => {
+contract('EternalStorageProxy with generic Eternal Storage', ([_, proxyOwner, tokenOwner, anotherAccount]) => {
```

Comment:
I think we can leave just `EternalStorageProxy` now

Reply:
file renames to EternalStorageProxy and `EternalStorageProxy with generic Eternal Storage` to `EternalStorageProxy`